test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -69,7 +69,7 @@
 INDENT_REGEX = re.compile(r'([\t]*)')
 TYPE_REGEX = re.compile(r'.*(if|or)(\s+[^"\']*|\s+)(?<!_)(?<!str\()type\([^)].*')
 SYS_EXIT_REGEX = re.compile(r'[^#]*sys.exit\s*\(.*')
-NO_LOG_REGEX = re.compile(r'(?:pass(?:[-_\s]?(?:word|phrase|wrd|wd)?)|secret|token|key)', re.I)
+NO_LOG_REGEX = re.compile(r'(?:pass(?!ive)(?:[-_\s]?(?:word|phrase|wrd|wd)?)|secret|token|key)', re.I)
```

Comment:
```suggestion
NO_LOG_REGEX = re.compile(r'(?:pass(?!ive)|secret|token|key)', re.I)
```
That part wasn't used anymore anyway...